Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -344,6 +345,12 @@ public struct PackageBuilder {
             }
         }
 
+        return modules
+    }
+
+    /// Fills the module dependencies delcared via targets in manifest.
+    private func fillDependencies(modules: [Module]) throws {
```

Comment:
Nit: `fillDependencies` doesn't seem like the right name (specifically "fill") but I don't have any good alternatives.  I'll comment again if I think of one ("establish" or "resolve", maybe).